tests/integration/_experimental/object_detection/test_evaluator_single_class_f1_optimal.py

```diff
@@ -0,0 +1,292 @@
+# Copyright 2021-2023 Kolena Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+from typing import List
+from typing import Tuple
+
+import pytest
+
+from kolena.workflow.annotation import LabeledBoundingBox
+from kolena.workflow.annotation import ScoredLabeledBoundingBox
+from kolena.workflow.plot import Curve
+from kolena.workflow.plot import CurvePlot
+from kolena.workflow.plot import Plot
+from tests.integration._experimental.object_detection.test_evaluator_single_class_fixed import (
+    assert_test_case_metrics_equals_expected,
+)
+from tests.integration._experimental.object_detection.test_evaluator_single_class_fixed import (
+    assert_test_case_plots_equals_expected,
+)
+from tests.integration._experimental.object_detection.test_evaluator_single_class_fixed import TEST_CASE
+from tests.integration._experimental.object_detection.test_evaluator_single_class_fixed import TEST_DATA
+from tests.integration.helper import fake_locator
+
+object_detection = pytest.importorskip("kolena._experimental.object_detection")
```

Comment:
On small thing we can add is put in `reason=xxx` so if developers are working locally and the tests are skipped, they will be more informed.